code/__DEFINES/sound_defines.dm

```diff
@@ -9,6 +9,7 @@
 #define CHANNEL_ENGINE 1017 // Engine ambient sounds
 #define CHANNEL_FIREALARM 1016 //fire alarm alarms
 #define CHANNEL_ASH_STORM 1015
+#define CHANNEL_RADIO_NOISE 1014
```

Comment:
Can you add a little comment for the channel's purpose

Reply:
Done.